index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="index.js" defer></script>
+    <style>            
+    @media screen and (max-width: 992px) {
+        body {
+        background-color: rgb(192, 192, 211);
+        }
+    }
```

Comment:
- [ ] [OPTIONAL] Great job @bobb-Rob, you could also that all styling should be done in the style.css, as this will not only make it professional but also make the code to be easily scalable by using same style for different pages.